torch/nn/modules/loss.py

```diff
@@ -1152,8 +1152,8 @@ class CTCLoss(_Loss):
         >>> ctc_loss = nn.CTCLoss()
         >>> log_probs = torch.randn(50, 16, 20).log_softmax(2).detach().requires_grad_()
         >>> targets = torch.randint(1, 21, (16, 30), dtype=torch.long)
-        >>> input_lengths = torch.full((16,), 50, dtype=torch.long)
-        >>> target_lengths = torch.randint(10,30,(16,), dtype=torch.long)
+        >>> input_lengths = torch.full((16,), 50, dtype=torch.long).toarray()
```

Comment:
hmm are you thinking about `tolist`? `toarray` doesn't seem to exist for me.

Reply:
Ouch.